lib/matplotlib/backends/backend_wx.py

```diff
@@ -864,16 +859,10 @@ def print_xpm(self, filename, *args, **kwargs):
     def _print_image(self, filename, filetype, *args, **kwargs):
         origBitmap = self.bitmap
 
-        l, b, width, height = self.figure.bbox.bounds
-        width = math.ceil(width)
-        height = math.ceil(height)
-
-        self.bitmap = wx.Bitmap(width, height)
-
+        self.bitmap = wx.Bitmap(*map(math.ceil, self.figure.bbox.size))
```

Comment:
Bit weird that you used `map` here, but the separate `width`/`height` properties in `wxagg`.